test/upgrade/README.md

```diff
@@ -44,7 +44,14 @@ To run the tests upgrading from the current source to the current source:
 To run just a particular test or tests:
 
 ```
-./mzcompose down -v ; ./mzcompose run default 'persistent-user-tables*'
+./mzcompose down -v ; ./mzcompose run default 'avro-ocf*'
+```
+
+If you are running a particular test that specifies the version, then you
+must include the `--min-version` flag with that version:
```

Comment:
It might be nice to add some extra context here, based on what you explained in https://github.com/MaterializeInc/materialize/pull/10872#issuecomment-1050034705

Also, it might worth filing a follow-up task to add flags that make it simpler to do local `upgrade/` testing. There are other issues, where if you are testing all the td files, locally you need `--most-recent <some number>` otherwise bash complains about the glob being too long now,

Reply:
Thanks for the feedback! I updated the comments with some more context, what do you think? 

Also I'm not personally familiar with any of the other issues, so I'm not sure I'm the best person to open a general issue, though I could open a specific one for this issue.